tools/cldr-code/src/main/java/org/unicode/cldr/util/VettingViewer.java

```diff
@@ -644,13 +666,33 @@ private void handleOnePath(String path) {
             if (!onlyRecordErrors) {
                 recordLosingDisputedEtc(path, voteStatus, missingStatus);
             }
+            updateVoterProgress(path, pathLevelIsTooHigh);
             if (specificSinglePath == null && !problems.isEmpty() && sorted != null) {
                 reasonsToPaths.clear();
                 R2<SectionId, PageId> group = Row.of(ph.getSectionId(), ph.getPageId());
                 sorted.put(group, new WritingInfo(ph, problems, htmlMessage));
             }
         }
 
+        private void updateVoterProgress(String path, boolean pathLevelIsTooHigh) {
+            if (voterProgress == null || voterId == 0) {
+                return;
+            }
+            if (pathLevelIsTooHigh && problems.isEmpty()) {
+                return;
+            }
+            voterProgress.incrementVotablePathCount();
+            if (userVoteStatus.userDidVote(voterId, cldrLocale, path)) {
+                voterProgress.incrementVotedPathCount();
+            }
+            /*
+             * Else, path is votable but the user hasn't voted on it yet.
+             * We could add path to a list of such paths and present it to the user,
+             * maybe as another category in the Dashboard, related to Missing, yet
+             * different since Missing is per organization, and this is per user.
+             */
```

Comment:
New category could be Abstain -- see https://unicode-org.atlassian.net/browse/CLDR-14047